src/main/java/org/spongepowered/common/event/tracking/phase/packet/PacketPhaseUtil.java

```diff
@@ -97,7 +97,7 @@ public static void validateCapturedTransactions(int slotId, Container openContai
     }
 
     public static void handlePlayerSlotRestore(EntityPlayerMP player, ItemStack itemStack, EnumHand hand) {
-        if (itemStack == null || itemStack == ItemTypeRegistryModule.NONE) {
+        if (itemStack.isEmpty() || itemStack == ItemTypeRegistryModule.NONE) {
```

Comment:
@Faithcaio 

Since Mojang added ItemAir, I think it would make the implementation much cleaner to just outright remove our concept of a NONE stack and simply use theirs.

@bloodmc @gabizou Your comments are also appreciated.